src/status_im/utils/ethereum/ens.cljs

```diff
@@ -18,98 +18,114 @@
 (def default-address "0x0000000000000000000000000000000000000000")
 (def default-key "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")
 
-(defn namehash [s]
-  (ethereum/normalized-address (if (string/blank? s)
-                                 default-namehash
-                                 (let [[label remainder] (-> s
-                                                             string/lower-case
-                                                             (string/split #"\." 2))]
-                                   (ethereum/sha3 (+ (namehash remainder)
-                                                     (subs (ethereum/sha3 label) 2))
-                                                  {:encoding "hex"})))))
+(defn namehash
+  [s]
```

Comment:
why do you move args down? i return it back every time :) we need a consensus

Reply:
we should always put they down, easier to add a docstring, easier to keep the line length limit